Dockerfile

```diff
@@ -1,20 +1,26 @@
 FROM node:13 as builder
 
-COPY package* ./
+COPY package.json yarn.lock ./
 RUN yarn
 
 COPY src ./src
 COPY tsconfig.json tsconfig.json
 
-RUN ./node_modules/.bin/tsc || true
+RUN yarn
+RUN yarn build || true
+
+###############################################################
 
 FROM node:13 as installer
 
 ENV NODE_ENV production
-COPY package.json package.json
+COPY package.json yarn.lock ./
```

Comment:
We might want to make sure this is installing production only or not.

Reply:
@XVincentX Sorry, I may not understand what change you'd like me to apply.
The line immediately below triggers a `RUN yarn --production`

FWIW, those changes have been greatly inspired from https://github.com/stoplightio/prism/blob/master/Dockerfile

In order to better align with the source, I've applied a minor fixup to that file.

Reply:
Then it probably works, I've been verifying every step in Prism.